### PR TITLE
Set up front end context and login

### PR DESCRIPTION
**Pull request by fluturecode**
Added axios and boostrap. Set up app.jsx, app.context, login.jsx.

### CODE REVIEW

File: client/sample.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_SERVER_URL=your servers address
```
Comment (Stommy123) [inline client/sample.env:1]: I don't think this env var is needed. In your `package.json` you have a proxy to your backend already. Meaning any request that you prefix with a forward slash `/`, will have that proxy prepended 

for example
```
axios.get('/users')
```
is actually going to make a request to `http://localhost:8080/user`

for reference
https://github.com/wyncode/C37_temp_housing/blob/0aee90c05250136e5442f880fa31b7900e8ea381/client/package.json#L16

File: client/src/components/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const Login = () => {
+  const { setUser, setLoggedIn } = useContext;
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const logIn = async (email, password, e) => {
+    e.preventDefault();
+    await axios({
+      method: 'POST',
+      url: `${process.env.REACT_APP_SERVER_URL}/users/login`,
```
Comment (Stommy123) [inline client/src/components/Login.jsx:14]: then can just be `'/users/login'`

Refer to comment above

File: client/sample.env (hunk 1 above)
Comment (fluturecode) [reply client/sample.env:1]: Ok understood. Will remove it now.

File: client/src/components/Login.jsx (hunk 2 above)
Comment (fluturecode) [reply client/src/components/Login.jsx:14]: got it!

File: package.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "dependencies": {
     "@sendgrid/mail": "^7.2.0",
+    "axios": "^0.19.2",
```
Comment (Stommy123) [inline package.json:8]: I don't see `axios` being used anywhere in your server. 

If im not mistaken this is only needed in your `client/package.json`.

Comment (fluturecode) [reply package.json]: Yes that was a mistake. Removed axios from backend.

**Pull request merged**